editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -702,7 +702,7 @@ void CanvasItemEditor::_find_canvas_items_in_rect(const Rect2 &p_rect, Node *p_n
 	}
 
 	CanvasItem *ci = Object::cast_to<CanvasItem>(p_node);
-	Node *scene = EditorNode::get_singleton()->get_edited_scene();
+	Node *const scene = EditorNode::get_singleton()->get_edited_scene();
```

Comment:
```suggestion
	Node *scene = EditorNode::get_singleton()->get_edited_scene();
```
Don't see any need for this pointer to be constant, we don't do this generally, and especially with a cleanup let's avoid unnecessary noise